vm/src/sys/audio.rs

```diff
@@ -1,9 +1,12 @@
+#[cfg(feature = "sdl")]
```

Comment:
It must be possible to conditionally include modules instead of adding conditional compilation to everything.

Reply:
Not that I know of.

Reply:
At the top of the file it only works on the following line. You can also not do ranges at the top level. 

The “sdl” features adds SDL support. Naming it “headless” is the opposite of what it does ?

Reply:
Again. The Rust way of doing things would be to put the headless core into a lib crate and the syscall into an app crate.

Reply:
Directives with `#!` apply to the whole file, as opposed to just `#`.

Reply:
Oh, I did not know that, nice! Updated the PR.

Reply:
But the question still stands. How can people like me use ```vm``` inside their projects as it is not a ```lib``` but a ```bin``` ?

I would need to copy the files out of your repo and include it directly in my project....

Reply:
Looks like there might be a way to create a workspace and have multiple crates inside, but at the moment, I don't wan to deal with the added complexity.

Reply:
Ok no problem. I have all my projects inside workspaces. Really convenient.